src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -877,6 +877,17 @@ std::string primitive_inst::generic_to_string(program_node const& node, const ch
     return primitive_description.str();
 }
 
+std::vector<layout> primitive_inst::forward_input0_shape(const kernel_impl_params& impl_param) {
```

Comment:
Could you use template ShapeType? Same as in the calc_output_layouts()

Reply:
Updated